vumi_http_proxy/test/helpers.py

```diff
@@ -1,14 +1,18 @@
 from twisted.web import server, resource
+from twisted.web.client import ResponseDone
+from twisted.web.http_headers import Headers
 from twisted.internet import reactor
-from twisted.internet.defer import inlineCallbacks, returnValue
+from twisted.internet.defer import inlineCallbacks, returnValue, succeed
 from twisted.internet.endpoints import serverFromString
-
+from twisted.python.failure import Failure
 
 DEFAULT_TIMEOUT = 2
 
+hostnames = {'zombo.com': '69.16.230.117',
+             'zombie.com': '66.96.162.142'}
```

Comment:
People often format dictionaries like this as:
```python
hostnames = {
    'zombo.com': '69.16.230.117',
    'zombie.com': '66.96.162.142',
}
```
(also not a very important comment).